code/game/turfs/space/space.dm

```diff
@@ -15,19 +15,16 @@
 /turf/space/New()
 	if(!istype(src, /turf/space/transit))
 		icon_state = "[((x + y) ^ ~(x * y) + z) % 25]"
-	if(config)
-		if(config.starlight)
-			update_starlight()

```

Comment:
it's already done in init_lighting() which is called on server start and by ChangeTurf() so this was redundant
pap put it here in the first PR but forgot to put it in init_lighting(), then whoever put it there forgot to remove this one